backend/search/episodes.go

```diff
@@ -4,9 +4,12 @@ import (
 	"context"
 	"fmt"
 	"github.com/bcc-code/brunstadtv/backend/common"
+	"os"
 	"strconv"
 )
 
+var imageCDNDomain = os.Getenv("IMAGE_CDN_DOMAIN")
```

Comment:
Can we get this into `cmd`? The main reason why I packaged all of them together is to avoid sudden surprises about env vars you knew noting about being read deep in the code.

Reply:
No very clear way to do this, but config.ImageCDNDomain should override the use of this now (only used when retrieving from database)